tests/test_corpora.py

```diff
@@ -38,10 +38,26 @@ def build_index_corpus(directory):
     return quantgov.corpora.IndexDriver(str(index_path))
 
 
+def build_s3_corpus(directory):
+    rows = []
+    for letter, number, path in (
+            ('a', '1', 'quantgov_tests/first.txt'),
+            ('b', '2', 'quantgov_tests/second.txt')
+    ):
+        rows.append((letter, number, path))
+    index_path = directory.join('index.csv')
+    with index_path.open('w', encoding='utf-8') as outf:
+        outf.write(u'letter,number,path\n')
```

Comment:
Should we be using the `csv.writerows` method for this? Or is this a more efficient way for testing purposes? @OliverSherouse

Reply:
Not more efficient, particularly, though not really a problem, either.